### DISCUSSION TITLE
Cannot uninstall raspap via installer script

### DISCUSSION THREAD

**Discussion by mcgyver83** [RaspAP/raspap-webgui#1492] [Q&A]
Hi!
I want to start from scratch an uninstall RaspAP.
I tried 
`pi@raspberrypi:~ $ curl -sL https://install.raspap.com | bash -s -- --uninstall`
but after some second nothing happens, no output.

**Comment by billz** [comment]
The installer will fail silently if you don't have a connection. Drop the `-s` (silent) parameter to see if you have internet:

```
curl -L https://install.raspap.com | bash
```

**Comment by Loggsie** [comment]
Command line 

cd /var/www/html

installers/uninstall.sh

**Reply by mcgyver83** [reply]
I got it.
Sadly I tried above command,  accepted all request and later I realised that I accepted to uninstall also "packages" used by others on my system (for a reason also "openmediavault-xxxx" packages where removed.

I got prompted so it's my fault, of course.

**Reply by billz** [reply]
The uninstaller will only remove packages installed/used by RaspAP, but there may of course be overlap with other software.

https://github.com/RaspAP/raspap-webgui/blob/bb23dc79f699c1cffffd673b410133ed8c005b87/installers/uninstall.sh#L226
